main.py

```diff
@@ -0,0 +1,50 @@
+from whetstone import Whetstone
+import json
+from os import path, makedirs
+
+
+def extract(api, endpoint, tag=False):
```

Comment:
Can you rename the function to something that makes it more clear what this is doing?

Reply:
Agreed. This has been deprecated in #2.